src/main/java/nextstep/subway/application/SectionService.java

```diff
@@ -43,6 +43,16 @@ public List<SectionResponse> retrieveSectionsByLine(Long lineId) {
         List<Section> sections = findLineById(lineId).getSections().getSectionList();
         return sections.stream().map(section -> SectionResponse.of(section)).collect(Collectors.toList());
     }
+    @Transactional
+    public SectionResponse deleteStationById(Long lineId, Long stationId) {
+        Line line = findLineById(lineId);
+        Station deleteStation = findStationById(stationId);
+
+        Section deletedSection = line.deleteStation(deleteStation);
+        sectionRepository.delete(deletedSection);
```

Comment:
```suggestion
        line.deleteStation(deleteStation);
```
`orphanRemoval` 옵션을 true로 주면 객체지향적으로 작성할수 있어요. 😀

Reply:
감사합니다!!
JPA의 옵션 정보들이 아직 익숙하지 않았는데 덕분에 한 개 더 학습 할 수 있었습니다. 🙏🙏